create_remote_nodes.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
+set -xe
 
+source logging.sh
 source common.sh
 source ocp_install_env.sh
 
 namespace=$1
 if [ $namespace == "" ]; then
-	echo "Error: Insufficient arguments. Please specify the namespace"
-	echo "Usage: $0 <namespace>"
-	exit 1
+	namespace=openshift-machine-api
```

Comment:
This could be simplified to e.g `namespace=${1:-openshift-machine-api}

Reply:
Cool. This looks amazing. I'll update it.